docs/modules/deploy/pages/deploying-with-operator.adoc

```diff
@@ -132,13 +132,55 @@ You can configure the `expose-externally` field with the following properties:
 ** `NodePortNodeName`: Hazelcast clients are inside the same network as the Kubernetes nodes, so clients can access a member by the hostname of its Kubernetes node.
 ** `LoadBalancer`: Hazelcast clients are outside the Kubernetes network and can access members via the the external IP assigned to the LoadBalancer service.
 
+=== Start Management Center
+
+You can monitor the Hazelcast cluster by starting the Management Center. You can create `ManagementCenter` Custom Resource file as `management-center.yaml`.
+
+[source,yaml]
```

Comment:
```suggestion
[source,yaml,subs="attributes+"]
```

Reply:
Sure, I will check and update them accordingly.